manage_proj_edit_page.php

```diff
@@ -492,11 +499,25 @@
 			<input type="submit" name="copy_to" class="btn btn-sm btn-primary btn-white btn-round" value="<?php echo lang_get( 'copy_versions_to' ) ?>" />
 		</fieldset>
 	</form>
-	</div>
+				</div>
+				<div class="pull-right">
+    <form id="manage-project-version-filter" method="post" action="manage_proj_edit_page.php?project_id=<?php echo $f_project_id ?>" class="form-inline">
+        <fieldset>
+			<input type="hidden" class="form-control input-sm" name="project_id" value="<?php echo $f_project_id ?>" />
```

Comment:
You're providing the project_id both here and in the form's action property, that's redundant